Remotion/Web/Development.WebTesting/Utilities/ProcessUtils.cs

```diff
@@ -143,7 +143,16 @@ public static void GracefulProcessShutdown ([NotNull] Process process, TimeSpan
       }
 
       // Force closing the process
-      process.Kill();
+      try
+      {
+        process.Kill();
+      }
+      catch (Win32Exception)
+      {
+        // Thrown if the .Kill() method is called while the process is currently terminating.
+        // https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.process.kill#remarks
+        // This exception can be swallowed safely, as the process that should be killed is already terminating.
+      }
 
       if (process.WaitForExit (timeoutInMilliseconds))

```

Comment:
Hmm...I think if the process is already dead at this point, we might get another exception (see docs). this one should be easy to test with a debugger to delay the call to WaitForExit until the process is gone. don't know if the currently being-killed process will also throw, same as in line 148, though, or if it's just a missing process that will throw. =-> check docs, do trials, make safe.

Reply:
That's an issue we must have met before the removal of this line: https://github.com/re-motion/Trunk-SVN2Git/pull/113/files#diff-3569e8db6d4a2bdf25ec2682ebd29c24L127

Pretty sure this is a non-issue.

Reply:
Yep, tested while asserting that the process has exited.